ecosystem/sep-0031.md

```diff
@@ -7,36 +7,47 @@ Title: Direct Payments
 Author: SDF
 Status: Active
 Created: 2020-04-07
-Updated: 2020-12-15
-Version 1.2.0
+Updated: 2021-04-20
+Version 1.2.1
 ```
 
 ## Simple Summary
 
-This SEP defines a protocol for enabling direct fiat to fiat payments between two financial accounts that exist outside of the Stellar network. The payments are facilitated by two anchors.
+This SEP defines a protocol for enabling payments between two financial accounts that exist outside the Stellar network.
 
 ## Abstract
 
-This proposal facilitates the ability for anchors to build a rail between two regions, allowing end users to send fiat from one bank account directly into another end user's bank account.  In this flow, neither user needs to deal with the Stellar network as the two anchors take care of everything for them.
+The entities involved in a transaction are:
 
-## Example
+- A **Sending Client**: The owner of the origin financial account.
+- A **Sending Anchor**: The business offering outbound payment services, who must have a business relationship with the Receiving Anchor.
+- A **Receiving Anchor**: The business offering inbound payment processing.
+- A **Receiving Client**: The owner of the destination financial account.
 
-Alice in Nigeria wants to send money to Bob in Europe. Alice signs up with NigeriaPay to make this payment to send money directly into Bob’s bank account. Bob doesn’t need to do anything, or know anything about this payment, besides letting Alice know what his bank account information is. Alice only needs to deal with her anchor (NigeriaPay).
+At a high level, the following steps are performed to complete a transaction:
 
-NigeriaPay will utilize its European rail, enabled with EuroPay Anchor service, to move the money to EuroPay in order to deposit it into Bob’s bank account.
+1. The Sending Client sends funds from a financial account to the Sending Anchor
+2. The Sending Anchor sends the associated amount of funds to the Receiving Anchor's Stellar account
+3. The Receiving Anchor sends funds to the Receiving Client's financial account
```

Comment:
💡 Is it true that steps 1 and 3 are always off-chain, and step 2 is always on-chain? It might be worth calling out where the Stellar is and where it isn't in these steps. 

I also think it is worth being specific that the sending client will send funds from their financial account rather than a financial account. Money transfer businesses, in my experience, discourage and avoid customers sending funds from other peoples accounts.

```suggestion
1. The Sending Client sends funds from their financial account to the Sending Anchor's financial account (off-Stellar)
2. The Sending Anchor sends the funds to the Receiving Anchor's Stellar account (on-Stellar)
3. The Receiving Anchor sends funds to the Receiving Client's financial account (off-Stellar)
```

Reply:
`2` is always on Stellar and `3` is always off Stellar, but `1` could theoretically be either.

- If the Sending Client wanted to send funds from their Stellar account to a Receiving Client's Stellar account, any wallet can do that.
- If the Sending Client wanted to send funds from their off-chain account to a Receiving Client's Stellar account, only one anchor is needed and therefore this case isn't relevant to this protocol.
- If the Sending Client wanted to send funds from their on-chain account to a Receiving Client's off-chain account, its not any different from an off-chain to off-chain account. The Sending Anchor provides an interface that allows the Sending Client to provide funds.

Reply:
Got it, makes sense, so like:

```suggestion
1. The Sending Client sends funds from their financial account to the Sending Anchor's financial account (off/on-Stellar)
2. The Sending Anchor sends the funds to the Receiving Anchor's Stellar account (on-Stellar)
3. The Receiving Anchor sends funds to the Receiving Client's financial account (off-Stellar)
```